net-misc/casync/casync-2_p20190226.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit meson
+
+COMMIT="095cb73c2efb625e7a5b9460e93e30565b4f87a0"
+DESCRIPTION=" Content-Addressable Data Synchronization Tool"
+HOMEPAGE="https://github.com/systemd/casync"
+SRC_URI="https://github.com/systemd/casync/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+fuse +udev man selinux test"
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+RDEPEND="
```

Comment:
Slot operators if it links to stuff?

Reply:
I'm sorry if this wasn't clear enough. I was talking of `:=` slot operator which you use when the package links to the library, not `:0` slots when the package is not slotted.